subsys/fs/Kconfig

```diff
@@ -54,13 +54,21 @@ config FILE_SYSTEM_INIT_PRIORITY
 config FILE_SYSTEM_SHELL
 	bool "File system shell"
 	depends on SHELL
-	depends on HEAP_MEM_POOL_SIZE > 0
 	help
 	  This shell provides basic browsing of the contents of the
 	  file system.
 
 if FILE_SYSTEM_SHELL
 
+config FILE_SYSTEM_SHELL_MOUNT_COMMAND
```

Comment:
Hi. Can you add additional commit to this PR mentioning the addition of the Kconfig to the Release notes? Thanks.

Reply:
@de-nordic Rebased and mentioned in 4.0 release notes as requested.